### ISSUE TITLE
the following ctas(button) on the nvda website: 'Free Download' and 'Donate' are orange(bg) with white(fg) text. Would this fail color contrast compliant test?

### ISSUE THREAD

**Issue by ronra** [nvaccess/nvda#8478]
### Steps to reproduce:
Go to nvaccess site and you'll see the following cta(buttons): 'Donate' and 'Free Download' on the page.
### Actual behavior:
Color contrast combination: orange background with white text would fail the tool (which would be webaim color contrast)
### Expected behavior:
The question is regarding the site itself for nvda. Are ctas required to meet color contrast compliance in regards to WCAG 2.0?

### System configuration:
N/A
#### NVDA Installed/portable/running from source:
N/A
#### NVDA version:
N/A
#### Windows version:
Windows 10
#### Name and version of other software in use when reproducing the issue:
N/A
### Other questions:
N/A
#### Does the issue still occur after restarting your PC?
N/A
#### Have you tried any other versions of NVDA?
N/A

**Comment by josephsl** [comment]
CC @feerrenrut

**Comment by feerrenrut** [comment]
I think this does fail the compliance test, and is something that NV Access would like to address. Thanks for pointing it out. 

These colours match the branding and logo and so we are hesitant to change them. WCAG recommends using a halo to improve the contrast around the text in these buttons.

From the definition of "contrast ratio" in the [glossary of WCAG2.1](https://www.w3.org/TR/WCAG21/#glossary)

> NOTE
> When there is a border around the letter, the border can add contrast and would be used in calculating the contrast between the letter and its background. A narrow border around the letter would be used as the letter. A wide border around the letter that fills in the inner details of the letters acts as a halo and would be considered background.

Some intereseting discussion on this approach when others have encountered this issue: https://github.com/GoogleChrome/accessibility-developer-tools/issues/302
https://github.com/CodeforAustralia/school-finder/issues/191

The current "Donate" button looks like this:
![image](https://user-images.githubusercontent.com/3491346/43697396-2e819f68-9983-11e8-9ebc-861049353c8d.png)

Using `text-shadow: 0 0 0.001em black;` to make the text look like the following:
![image](https://user-images.githubusercontent.com/3491346/43697361-da0f1d8e-9982-11e8-9071-5c678a6d1cdf.png)

The current banner text:
![image](https://user-images.githubusercontent.com/3491346/43697428-63a4b91e-9983-11e8-8b19-8046db54cd43.png)

And with the shadow applied.
![image](https://user-images.githubusercontent.com/3491346/43697449-7f3a0b5c-9983-11e8-8d99-e3d852e7ea0e.png)

Further down the page are prices for featured products, these are orange on white:
![image](https://user-images.githubusercontent.com/3491346/43697563-498c720a-9984-11e8-8d1f-9d76f1a37111.png)
And with the shadow applied.
![image](https://user-images.githubusercontent.com/3491346/43697571-58745ad0-9984-11e8-8302-778a4023a583.png)

**Comment by feerrenrut** [comment]
Quentin is going to have a play with the colours, and then Reef will update the website.

**Comment by feerrenrut** [comment]
We have added text shadows to improve the contrast on these items, these will still fail many of the automated tools (which don't take the text shadow into account), however we believe that we have addressed the contrast requirements for these buttons.
